sys/malloc_thread_safe/malloc_wrappers.c

```diff
@@ -45,28 +63,53 @@ void __attribute__((used)) __wrap_free(void *ptr)
 
 void * __attribute__((used)) __wrap_calloc(size_t nmemb, size_t size)
 {
+    uinttxtptr_t pc;
+    if (IS_USED(MODULE_MALLOC_TRACING)) {
+        pc = cpu_get_caller_pc();
+    }
```

Comment:
```else (void)pc;``` ?

Reply:
Should not be an issue due to not using preprocessor conditionals.